### DISCUSSION TITLE
Disabling auto onboarding for our organisation

### DISCUSSION THREAD

**Discussion by anthr76** [renovatebot/renovate#26254] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

None

### Describe the problem

Hi, we would like to enable Renovate for all repo's in our organisation ([http://github.com/coreweave](https://github.com/coreweave)), but this would lead to a lot of unwanted onboarding PR's being created.

I noticed in https://github.com/renovatebot/renovate/discussions/17134 / https://github.com/renovatebot/renovate/discussions/18432 and similar topics that we should request a modification here.

Could you please set the "onboarding":false setting for the Sectigo GitHub organisation so that Renovate will run only on those repo's with a pushed configuration but without creating PR's for the ones that don't have a config?

Thank you very much for your assistance!

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment, marked as answer]
When you install an org through the github app today it will take you to developer.mend.io and ask you to pick the org default:
- Disable
- Silent scan
- Full

You can pick either of the first two and you won't get automatic onboarding. If you pick the third option then you can also decide after that whether to onboard or not.

Create a test org with one repo and try it out there first to be sure, if you like.

**Reply by anthr76** [reply]
Understood! Thanks :)
